api/validate.py

```diff
@@ -0,0 +1,39 @@
+import json
+
+import urllib.request
+from json import JSONDecodeError
+from urllib.error import URLError
+
+from flask import Blueprint, request, jsonify, Response
+from validation.validator import Validator
+
+validate_blueprint = Blueprint('validate', __name__)
+
+validator = Validator()
+
+
+@validate_blueprint.route('/validate', methods=['POST'])
+def validate_schema():
+    data = None
+
+    values = request.form
+    if 'url' in values:
```

Comment:
It would be better if the url functionality worked by a GET request `url` query param. It would allow quick testing of a schema via a simple link rather than having to construct a POST request.